docs/packages/connect/methods/init.md

```diff
@@ -0,0 +1,25 @@
+## Init
+
+Initiate TrezorConnect
+
+```javascript
+const result = await TrezorConnect.init(params);
+```
+
+### Params
+
+[init type](https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/types/api/init.ts)
+
+-   `manifest` - _required_ `Object` of type `[Manifest]`(https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/types/settings.ts#L3),
+-   `connectSrc` - _optional_ `string`. Set custom src for TrezorConnect iframe.
+-   `debug` - _optional_ `boolean`, default `false`. Prints debug logs.
+-   `transportReconnect` - _optional_ `boolean`, default `true`. If transport dies during lifecycle if application, this field decides whether TrezorConnect tries to reestablish connection with transport layer.
+-   `pendingTransportEvent` - _optional_ `boolean`, default `true`. Postpone emitting TRANSPORT.START event to the moment when connected device becomes available.
+-   `interactionTimeout` - _optional_ `number`, default `600`. Time in milliseconds after which popup automatically closes.
```

Comment:
i think this may be seconds not milliseconds

Reply:
you are right, my bad, will fix